### ISSUE TITLE
parameter-list-spacing conflicts with itself and other rules

### ISSUE THREAD

**Issue by vanniktech** [pinterest/ktlint#1509]
After updating to Version 0.46.0, the following line gets flagged when using the experimental flag:

```kotlin
private val isSelected: Boolean, // This is only here to invalidate all the other items.
```

> :30:35: Unexpected whitespace (experimental:parameter-list-spacing)

When correcting it:

```diff
-private val isSelected: Boolean, // This is only here to invalidate all the other items.
+private val isSelected: Boolean,// This is only here to invalidate all the other items.
```

And running ktlint again, it conflicts with:

- experimental:parameter-list-spacing
- comma-spacing
- comment-spacing

```
30:34: Whitespace after ',' is missing (experimental:parameter-list-spacing)
30:35: Missing spacing after "," (comma-spacing)
30:35: Missing space before // (comment-spacing)
```

So the initial error seems to be a false positive.

**Comment by paul-dingemans** [comment]
Can you please give a bit more context in which the line is used? I can not reproduce the problem. Also, specify the relevant `.editorconfig` settings.

**Comment by vanniktech** [comment]
Editorconfig:

```
[*.{kt,kts}]
indent_size=2
continuation_indent_size=2
insert_final_newline=true
disabled_rules=annotation,argument-list-wrapping,spacing-between-declarations-with-annotations,trailing-comma,filename
```

The file:

```kotlin
data class FormDateData(
  val localDate: LocalDate,
  private val isSelected: Boolean, // This is only here to invalidate all the other items.
)
```
